### PR TITLE
fix: updated TypeScript, C#, Go language names in codeGenerators.ts

### PR DESCRIPTION
**Pull request by jkaster**
Adopting the official spelling of "TypeScript"

### CODE REVIEW

**Review by josephaxisa — COMMENTED** [review]
LGTM but it seems like there are some missing dependencies based on the CI test results

**Review by josephaxisa — DISMISSED** [review]
The check failures are not from tests and are unrelated to your PR. Ship it!

**Review by joeldodge79 — DISMISSED** [review]
holy smokes! that these "unchanged file annotations" are so cool!

something is wonky with our CI checks though. as reported on the PR "Codegen CI / unit (pull_request)" was  "Skipped" but if you go to the "checks" tab of the PR you can see it got assigned twice - once was skipped (this is a bug do to cla-bot labeling all our PRs now, need to figure that out) and one failed which caused the annotations. So CI has actually [failed for this PR](https://github.com/looker-open-source/sdk-codegen/runs/2396623399?check_suite_focus=true)

clearly there's still work to be done to have PRs show the true check status...

File: packages/sdk-codegen/src/codeGenerators.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ export const Generators: Array<IGeneratorSpec> = [
   {
     factory: (api: ApiModel, versions?: IVersionInfo) =>
       new GoGen(api, versions),
-    language: 'Golang',
+    language: 'Go',
```
Comment (joeldodge79) [inline packages/sdk-codegen/src/codeGenerators.ts:91]: These seem like potentially user facing fixes? Maybe consider `s/chore/fix/` in the PR title? But totally your discretion

File: packages/sdk-codegen/src/codeGenerators.spec.ts [hunk 2]
```diff
@@ -62,18 +63,18 @@ describe('generator factory', () => {
     it('returns generator by file extension', () => {
       let actual = findGenerator('.ts')
       expect(actual).toBeDefined()
-      expect(actual?.language).toEqual('Typescript')
+      expect(actual?.language).toEqual(typeScript)
```
Comment (joeldodge79) [inline packages/sdk-codegen/src/codeGenerators.spec.ts:66]: I'm in favor of your change, but maybe for consistency re-do the `"Csharp"` and `"Kotlin"` magic strings in the `describe` block too?

File: packages/sdk-codegen/src/codeGenerators.ts (hunk 1 above)
Comment (jkaster) [reply packages/sdk-codegen/src/codeGenerators.ts:88]: Sure, it can be a `fix`

File: packages/sdk-codegen/src/codeGenerators.spec.ts (hunk 2 above)
Comment (jkaster) [reply packages/sdk-codegen/src/codeGenerators.spec.ts:68]: Done

**Review by joeldodge79 — APPROVED** [review]

**Pull request merged**